qiskit/transpiler/preset_passmanagers/default.py

```diff
@@ -98,6 +100,9 @@ def default_pass_manager_simulator(basis_gates):
         PassManager: A passmanager that just unrolls, without any optimization.
     """
     pass_manager = PassManager()
+
+    pass_manager.append(Decompose(InitializeGate))
```

Comment:
Shouldn't need this, because the `Unroller` pass will automatically decompose the initialize instruction if `initialize` is not in the backend's basis gates.

In fact we should not force decomposition here. Because simulators usually support direct initialization, which is much faster than decomposing.

Reply:
Fixed in cbc7fb5d